src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx

```diff
@@ -78,7 +91,20 @@ export default function VoteInformation({ proposal }: { proposal: ProposalDetail
         <Text color={theme.subText}>
           <Trans>Quorum Status</Trans>
         </Text>
-        <Text color={theme.text}>{proposal.vote_stats.quorum_status === 1 ? t`Reached` : t`Not Reached`}</Text>
+        {proposal.vote_stats.quorum_status === 1 ? (
+          <Text color={theme.text}>{t`Reached`}</Text>
+        ) : (
+          <MouseoverTooltip
+            text={`Total amount required: ${Math.floor(+totalAmountRequired.toFixed(0)).toLocaleString()} KNC`}
+            placement="bottom"
+            width="fit-content"
+          >
+            <Row width="fit-content" gap="6px" color={theme.warning}>
+              <WarningIcon size="16" solid />
+              <Text color={theme.warning} fontWeight={500}>{t`Not Reached`}</Text>
```

Comment:
<Trans>Not Reached</Trans>

Reply:
Yes?

Reply:
`<Trans>Not Reached</Trans>`

Reply:
https://lingui.dev/tutorials/react-patterns

Reply:
Fixed.